src/Server/RestHTTPRequestHandler.cpp

```diff
@@ -368,6 +368,11 @@ void RestHTTPRequestHandler::handleRequest(HTTPServerRequest & request, HTTPServ
         if (!authenticateUser(request_context, request, params, response))
             return; // '401 Unauthorized' response with 'Negotiate' has been sent at this point.
 
+        const auto & database = getDatabaseByUser(client_info.current_user);
+
+        if (!database.empty())
```

Comment:
If `user->database` mapping doesn't exist, we need throw

Reply:
fixed